docs/src/app/docs/layout.tsx

```diff
@@ -37,66 +37,57 @@ export default async function RootLayout({
   }));
 
   return (
-    <html lang="en">
-      <body>
```

Comment:
Despite the big diff the only change here _should_ be the removal of the html and body tags since we have a new layout higher up in the app directory.